f5_openstack_agent/lbaasv2/drivers/bigip/icontrol_driver.py

```diff
@@ -1666,3 +1677,9 @@ def trace_service_requests(self, service):
             fp.write(',')
             json.dump(service, fp, sort_keys=True, indent=2)
             fp.write(']')
+
```

Comment:
Maybe ESD, could be moved into the LBAAS builder, since it relies upon the ESD interface.  Having the lbaas_builder call back to icontrol_driver might not be the best flow of control.  Either have the lbaas_builder initialize the ESD and use it, or have icontrol_driver initialize the ESD and pass the esd object into the lbaas builder initialization.